### PR TITLE
fix: npn directory

### PR DESCRIPTION
**Pull request by frisonisland**
- put back original instructions but using local repo sources
- bump 4.1.1

Original PKGBUILD in 24.1.0 was appserver PKGBUILD: https://github.com/zextras/carbonio-core/pull/46/files#diff-3dabacea4af330e310d07293bb5a11f33b4d817f7a3a54cb5cfe4e34e6302ed8

### CODE REVIEW

**Review by keshavbhatt — APPROVED** [review]

File: appserver/PKGBUILD [hunk 1]
```diff
@@ -39,6 +39,7 @@ sha256sums=(
 
 package() {
 
+
```
Comment (M0Rf30) [inline appserver/PKGBUILD:42]: Remove extra space

File: appserver/PKGBUILD [hunk 2]
```diff
@@ -52,52 +53,70 @@ package() {
 
   install -D conf/antisamy.xml \
     "${pkgdir}/opt/zextras/conf/antisamy.xml"
+
```
Comment (M0Rf30) [inline appserver/PKGBUILD:56]: Remove extra space

**Review by M0Rf30 — CHANGES_REQUESTED** [review]
Remove alla unneded extra SpaceX and do a shfmt pass

**Review by drazen04 — APPROVED** [review]

File: appserver/PKGBUILD (hunk 2 above)
Comment (frisonisland) [reply appserver/PKGBUILD:54]: Check now, I've put a space between install for readability as I was not able to distinguish lines

**Review by sonersivri — APPROVED** [review]

**Review by M0Rf30 — APPROVED** [review]

**Pull request merged**